test/python/quantum_info/test_quaternions.py

```diff
@@ -11,14 +11,18 @@
 import numpy as np
 import scipy.linalg as la
 
-from qiskit.quantum_info.operators.quaternion import quaternion_from_euler
+from qiskit.quantum_info.operators.quaternion import quaternion_from_euler, Quaternion
+
 from qiskit.test import QiskitTestCase
 
 
 class TestQuaternions(QiskitTestCase):
     """Tests qiskit.quantum_info.operators.quaternion"""
 
     def setUp(self):
+        self.rndArray = np.array([0.5, 0.8, 0.9, -0.3])
```

Comment:
The convention we use for multi word variable names is with an underscore. In this case `random_array`.